block/stream/output/smart_contract_service.proto

```diff
@@ -158,4 +158,22 @@ message EthereumOutput {
      * This SHALL be a keccak256 hash of the ethereumData.
      */
     bytes ethereum_hash = 2;
+
+    oneof eth_result {
+        /**
+         * A result for an Ethereum Transaction executed as a call.
+         * <p>
+         * This field SHALL contain all of the data produced by the contract
+         * call transaction as well as basic accounting results.
+         */
+        proto.ContractFunctionResult ethereum_call_result = 3;
+
+        /**
+         * A result for an Ethereum Transaction executed as a create.
+         * <p>
+         * This field SHALL contain all of the data produced by the contract
+         * create transaction as well as basic accounting results.
+         */
+        proto.ContractFunctionResult ethereum_create_result = 4;
+    }
```

Comment:
This change is under some discussion in the Block Stream HIP, so @Nana-EC may wish to weigh in on it.
There are some other, related, changes to the Block Stream as well.

Reply:
Created [this issue](https://github.com/hashgraph/hedera-services/issues/16991) to address the suggestions in `hedera-services` repo. 👍